geoips/plugins/modules/readers/ahi_hsd.py

```diff
@@ -1200,8 +1213,11 @@ def call(
                 break
         if (not self_register) and (res not in gvars.keys() or not gvars[res]):
             LOG.info(
-                "We don't have geolocation information for {} for {} skipping {}".format(
-                    res, adname, chan
+                """We don't have geolocation information for {0} for {1} skipping
```

Comment:
Probably f strings broken up on multiple lines
f"line1"
"line2"
f"line3"

Reply:
fixed this